crates/blockchain/blockchain.rs

```diff
@@ -24,12 +30,20 @@ use ethrex_vm::{backends::BlockExecutionResult, get_evm_backend_or_default};
 //TODO: Implement a struct Chain or BlockChain to encapsulate
 //functionality and canonical chain state and config
 
+static GAS_COUNTER: AtomicU64 = AtomicU64::new(0);
```

Comment:
@JereSalo is creating a `blockchain` struct. We could wait for that so that we add this variable to there instead of having it as a global. Then we could also move the tokio task here, when creating the struct.

Reply:
This will be changed to no longer be a tokio task, but rather just a simple calculation pre and post execution, removing the global variable as well cc @Oppen

Reply:
ok, sounds good